test/unit/org/apache/cassandra/locator/ReplicaCollectionTest.java

```diff
@@ -182,7 +160,17 @@ private void assertSubList(C subCollection, int from, int to)
             }
         }
 
-        void testSubList(int subListDepth, int filterDepth, int sortDepth, int selectDepth)
+        private void assertSubSequence(Iterable<Replica> subSequence, int from, int to)
```

Comment:
Unused and also kind of weird (elementsEqual twice?)

Reply:
Ah, poor merge from a follow-up patch where I introduced filterLazily